TMSFTT/urls.py

```diff
@@ -21,6 +21,7 @@
 API_URLPATTERNS = [
     path('auth/', include('auth.urls')),
     path('training_event/', include('training_event.urls')),
+    path('training_program/', include('training_program.urls')),
```

Comment:
```suggestion
    path('training-program/', include('training_program.urls')),
```
I think dash is a better practice than underscore, and sorry for the bad example, I've fixed it.